### ISSUE TITLE
A11Y Changes -- Sprint 5

### ISSUE THREAD

**Issue by alexsoble** [18F/crt-portal#167]
## Changelog  

+ [X] If a user tries to go "Next" on the Protected Class page without selecting any checkboxes and has a screen reader turned on, they should hear an announcement about the error when the Protected-Class page re-loads
+ [x] The checkboxes seem to have two focus areas. **see checkbox note below**
+ [x] Remove extra space after protected class next button. Hopefully small, but the "next" button the protected class page seems to have a space included after it so it announces "next button space" on a screen reader when you use the up/down arrow keys. The Contact page button doesn't have this extra space.

**Comment by alexsoble** [comment]
Should be fixed by: https://github.com/usdoj-crt/crt-portal/pull/124.

**Comment by alexsoble** [comment]
@Jacklynn I know that you're in a Team Coffee related meeting right now, but this is ready for checking whenever! The fix is live on dev. We could also check in a pairing session sometime, no urgency here.

**Comment by Jacklynn** [comment]
The alert didn't read out to me on the dev site + safari + voiceover. Let me know if I checked in the wrong place

**Comment by alexsoble** [comment]
Notes from pairing on this with @Jacklynn and @LindsayYoung — we're going to try adding some JS that runs on page load and moves focus to any elements that have `role=alert`. Great suggestion from @LindsayYoung!

**Comment by LindsayYoung** [comment]
https://github.com/usdoj-crt/crt-portal/pull/131

**Comment by LindsayYoung** [comment]
@Jacklynn this is in dev, I reviewed locally, but would you like to give it a once over?

**Comment by Jacklynn** [comment]
I'm not sure why, but I couldn't get this to work reliably. If it was already showing and I hit next a 1-2 times it would announce, but 2/3 times it did not read the on first load (after refreshing the form).

**Comment by Jacklynn** [comment]
### Checkbox seems to have two focuses
- You can recreate this by using the arrow keys to navigate through the checkboxes instead of the tabbing. Expected behavior can be recreated here: https://www.w3.org/TR/wai-aria-practices/examples/checkbox/checkbox-1/checkbox-1.html
- Videos [1. voiceover of current experience and 2. on the w3 page (safari)](https://drive.google.com/drive/folders/1AcunqTidy7K16EtXaG1x7_3s3N1iP3V5?usp=sharing)

🎀*As Robert mentioned, screen reader users use different ways to navigate through a website, some because of workarounds because a website isn't working the way that they expect, but there are also novice screen reader users too, those who might be just learning, be using it later in life, or otherwise new to technology in general*

**Comment by lalitha-jonnalagadda** [comment]
@Jacklynn - since we have a new card for new issues, can you confirm UAT on this card and move this to done?

**Comment by Jacklynn** [comment]
I can take a look at this today!

**Comment by Jacklynn** [comment]
Noting that our 508 coordinator reviewed the checkbox page and no issues there so marking that as done. Will just check the extra space!

**Comment by Jacklynn** [comment]
@lalitha-jonnalagadda The next button is still reading a space afterwards. you can recreate this if you use the arrow keys to get to it with VO, but it doesn't read the space if you tab to it.

**Comment by Jacklynn** [comment]
Added last item to sprint 6 card https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/183
